packages/config/src/options.js

```diff
@@ -324,5 +324,20 @@ export function getNuxtConfig(_options) {
     bundleRenderer.runInNewContext = options.dev
   }
 
+  // Enable rate limit for dev mode
+  if (options.dev) {
+    if (typeof options.server.rateLimit === 'undefined') {
+      // It is unsual to request same resource more than 5 times in a 1sec!
+      options.server.rateLimit = {
+        windowMs: 1000,
+        max: 5,
+        keyGenerator: req => req.url,
+        onLimitReached: (req) => {
+          consola.warn('Too fast requests detected on path: ' + req.url)
```

Comment:
```suggestion
          consola.warn('Too many requests in a short time detected for path: ' + req.url)
```

Maybe?

Reply:
Agree with the typo change

Reply:
I suggest moving it to middleware defaults